src/main/java/com/google/cloud/tools/appengine/api/deploy/StageStandardConfiguration.java

```diff
@@ -16,45 +16,129 @@
 
 package com.google.cloud.tools.appengine.api.deploy;
 
+import com.google.common.base.Preconditions;
 import java.io.File;
 import javax.annotation.Nullable;
 
-/** Configuration for {@link AppEngineStandardStaging#stageStandard(StageStandardConfiguration)}. */
-public interface StageStandardConfiguration {
+/**
+ * Arguments needed to stage an App Engine standard environment application. Null return values
+ * indicate that the configuration was not set, and thus assumes the tool default value.
+ */
+public class StageStandardConfiguration {
```

Comment:
It seems like if FlexibleStaging is a builder, all configurations should be builders.

Reply:
done

Reply:
We can. My thought was the nullable fields that still have setters didn't need the builder, but as long as they're primarily for initializers we can put them all in the builder.

I'd rather not allow these fields to be null at all, but that's a more far reaching change and decision for another PR.